pastepwn/analyzers/__init__.py

```diff
@@ -2,33 +2,35 @@
 from .alwaystrueanalyzer import AlwaysTrueAnalyzer
 from .basicanalyzer import BasicAnalyzer
 from .bcrypthashanalyzer import BcryptHashAnalyzer
-from .md5hashanalyzer import MD5HashAnalyzer
-from .shahashanalyzer import SHAHashAnalyzer
 from .creditcardanalyzer import CreditCardAnalyzer
+from .dbconnstringanalyzer import DBConnAnalyzer
 from .genericanalyzer import GenericAnalyzer
+from .ibananalyzer import IBANAnalyzer
 from .mailanalyzer import MailAnalyzer
+from .mailpasswordanalyzer import MailPasswordAnalyzer
+from .md5hashanalyzer import MD5HashAnalyzer
 from .pastebinurlanalyzer import PastebinURLAnalyzer
+from .privatekeyanalyzer import PrivateKeyAnalyzer
 from .regexanalyzer import RegexAnalyzer
```

Comment:
The sorting could lead to circular imports but I will check that myself before merging - no worries :)